src/components/navigation/StepsNav.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="steps-nav">
+    <button class="steps-nav__button" :class="{ active: true }">
+      <p class="steps-nav__button-num" :class="{ active: true }">1</p>
+      Choose seats
+    </button>
+    <button class="steps-nav__button" :class="{ active: false }">
+      <p class="steps-nav__button-num" :class="{ active: false }">2</p>
+      Book tickets
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "StepsNav",
+  computed: {
+    //   WIP TO-DO
```

Comment:
With the approach I mentioned above, you could have this part controlled by the parent component, and `isActive` class being passed down as `props`. Then the `active` class could be defined by negating the prop. In that case, `isActive` could be also renamed to something like `isOnStepOne`.

```suggestion
<template>
  <div class="steps-nav">
    <button class="steps-nav__button" :class="{ active: isActive }">
      <p class="steps-nav__button-num" :class="{ active: isActive }">1</p>
      Choose seats
    </button>
    <button class="steps-nav__button" :class="{ active: !isActive }">
      <p class="steps-nav__button-num" :class="{ active: !isActive }">2</p>
      Book tickets
    </button>
  </div>
</template>

<script>
export default {
  name: "StepsNav",
  props: {
    isActive: {
       type: Boolean,
       required: true,
    }
```